### DISCUSSION TITLE
DTU ohne separates 2.4 GHz Modul??

### DISCUSSION THREAD

**Discussion by antonmeyer** [tbnobody/OpenDTU#1007] [Ideas]
Hallo zusammen,
ich frage mich warum man für die Kommunikation mit dem Wechselrichter nicht das 2.4 GHz Modul des ESP32 benutzt. Ist die API dafür nicht offengelegt?
esp-now ist ja auch so ein spezieller Modus. Und mit etwas Gefummel kann man sogar einen esp32 gleichzeitig im WLAN und im esp-now mode betreiben.
Ok, bei 2.50 € für das nrf24 module lohnt der Aufwand in der SW vermutlich nicht.
Aber man würde sich das Löten sparen ...

**Comment by Omega13x** [comment]
Ich vermute es hat was mit der Datenrate zu tun. Es funktionieren auch nur die "NRF24L01+", da die ohne "+" nicht die geringe Datenrate können.
![Zwischenablage01](https://github.com/tbnobody/OpenDTU/assets/123419851/a180e93b-de3d-4318-9bc9-00765740a4bb)
